wao/strategy_controller.py

```diff
@@ -1,14 +1,16 @@
 from wao.util import _perform_execute, _perform_back_test
 import threading
 from wao.config import Config
+from wao.notifier import send_start_deliminator_message
 import time
 
 
 class StrategyController:
     romeo_pool = {}
+    brain = ''
```

Comment:
remove redundant variable - according to the comment on line 39 this variable is redundant